contracts/merkle-tree/MerkleTree.sol

```diff
@@ -46,14 +46,14 @@ contract MerkleTree is MiMC {
     /**
     These events are what the merkle-tree microservice's filters will listen for.
     */
-    event NewLeaf(uint leafIndex, uint leafValue, uint root);
-    event NewLeaves(uint minLeafIndex, uint[] leafValues, uint root);
+    event NewLeaf(uint indexed leafIndex, uint indexed leafValue, uint indexed root);
+    event NewLeaves(uint indexed minLeafIndex, uint[] leafValues, uint indexed root);
 
     // event Output(uint[] input, uint[] output, uint prevNodeIndex, uint nodeIndex); // for debugging only
 
-    uint public zero = 0;
-    uint public treeHeight = 32; //
-    uint public treeWidth = 2 ** treeHeight;
+    uint public constant zero = 0;
+    uint public constant treeHeight = 32;
+    uint public constant treeWidth = uint64(2 ** treeHeight);
```

Comment:
Reject the change to cast to a uint64.  Accidentally left this in from when I was testing treeWidth as a uint64.  Make sure to as constant though.